components/Theme/ThemeProvider.tsx

```diff
@@ -0,0 +1,14 @@
+"use client"; //Unsure if required in current next version
```

Comment:
This isn't needed in this version! But that's on my plate to upgrade.

Reply:
Noted! Will address